tests/pds/pds_storagesize_increase_test.go

```diff
@@ -48,11 +49,23 @@ var _ = Describe("{ResizeStorageAndRestoreWithVariousFSandRepl}", func() {
 			newResourceTemplateID    string
 			newStorageTemplateID     string
 			updatedPvcSize           uint64
+			beforeResizePodAge       float64
 		)
+		credName := targetName + pdsbkp.RandString(8)
+		bkpClient, err = pdsbkp.InitializePdsBackup()
+		log.FailOnError(err, "Failed to initialize backup for pds.")
+		stepLog := "Create AWS S3 Backup target."
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			bkpTarget, err := bkpClient.CreateAwsS3BackupCredsAndTarget(tenantID, fmt.Sprintf("%v-aws", credName), deploymentTargetID)
+			log.FailOnError(err, "Failed to create AWS backup target.")
+			log.InfoD("AWS S3 target - %v created successfully", bkpTarget.GetName())
+			bkpTargets = append(bkpTargets, bkpTarget)
+		})
 		pdsdeploymentsmd5Hash2 := make(map[string]string)
```

Comment:
rename pdsdeploymentsmd5Hash2 like before/after lets not use number, it improves code readability